cmd/collectors/zapi/plugins/quota/quota.go

```diff
@@ -172,42 +172,43 @@ func (my *Quota) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 				}
 
 				if attrValue := quota.GetChildContentS(attribute); attrValue != "" {
-					// Ex. InstanceKey: SVMA.vol1Abc.qtree1.5.disk-limit
-					instanceKey := vserver + "." + volume + "." + tree + "." + strconv.Itoa(quotaIndex) + "." + attribute
-					instance, err := my.data.NewInstance(instanceKey)
-
-					if err != nil {
-						my.Logger.Debug().Msgf("add (%s) instance: %v", attribute, err)
-						return nil, err
-					}
-
-					my.Logger.Debug().Msgf("add (%s) instance: %s.%s.%s", attribute, vserver, volume, tree)
-
 					qtreeInstance := data.GetInstance(tree + "." + volume + "." + vserver)
-					for _, label := range my.data.GetExportOptions().GetChildS("instance_keys").GetAllChildContentS() {
-						if value := qtreeInstance.GetLabel(label); value != "" {
-							instance.SetLabel(label, value)
+					if qtreeInstance.IsExportable() {
```

Comment:
minor, but this would be a little easier to read and diff if this was `if !qtreeInstance.IsExportable() {continue}` and then the happy path doesn't change